plugins/modules/cloud/misc/proxmox.py

```diff
@@ -115,6 +115,10 @@
     description:
       - specifies network interfaces for the container. As a hash/dictionary defining interfaces.
     type: dict
+  features:
+    description:
+      - specifies a list of features to be enabled. For valid options, see https://pve.proxmox.com/wiki/Linux_Container#pct_options
+    type: str
```

Comment:
Since you say this should be a list (and not a string):
```suggestion
    type: list
    elements: str
    version_added: 2.0.0
```